pkgs/development/lua-modules/overrides.nix

```diff
@@ -76,6 +76,17 @@ with super;
     */
   });
 
+  ldbus = super.ldbus.override({
+    extraVariables = ''
```

Comment:
this is now a set, (check pkgs/development/interpreters/lua-5/build-lua-package.nix). If the doc is wrong tell me.

Reply:
Done. I'm actually using the other packages as examples to do this, what part of the documentation you're referring to?